src/Mail/SnapshotMonth.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Reviews\Mail;
+
+use Carbon\Carbon;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Tipoff\Locations\Models\Market;
+use Tipoff\Reviews\Models\Competitor;
+
+class SnapshotMonth extends Mailable
+{
+    use Queueable;
+    use SerializesModels;
+
+    public $competitors;
+    public $market;
+
+    /**
+     * Create a new message instance.
+     *
+     * @param Market $market
+     */
+    public function __construct(Market $market)
+    {
+        $this->market = $market;
+
+        $competitors = Competitor::where('market_id', $market->id)->get();
+        $this->competitors = $competitors->sortByDesc('reviews');
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        $execs = [
+            'kirk@thegreatescaperoom.com',
+            'julie@thegreatescaperoom.com',
+            'nicole@thegreatescaperoom.com',
+            'kelleigh@thegreatescaperoom.com',
+            'callcenter@thegreatescaperoom.com',
+            'scott@thegreatescaperoom.com',
+            'igug@thegreatescaperoom.com',
+        ];
+
+        $competitors = $this->competitors;
+        $market = $this->market;
+        $month = Carbon::now()->subMonth()->format('F');
+        $this->withSwiftMessage(function ($message) {
+            $message->getHeaders()->addTextHeader('tag', 'snapshotmonth');
+        });
+
+
+        if (isset($market->locations->first()->manager->email) && $market->locations->first()->manager->email !== $market->locations()->first()->contact_email) {
```

Comment:
@chx2 - I think your asking about `->manager->email` during tests.  Here's what I would do - these patterns `$market->locations->first()->manager->email` and `$market->locations()->first()->contact_email` occur a few times in this PR.  I would refactor by adding a `getManagerEmail()` and `getContactEmail()` to the Market model and also add a `getLocationEmails()` which returns the unique set.   

```
    public function getManagerEmail(): ?string
    {
        /** @var Location $location */
        if ($location = $this->locations->first()) {
            return $location->manager->email ?? null;
        }
        
        return null;
    }

    public function getContactEmail(): ?string
    {
        /** @var Location $location */
        if ($location = $this->locations->first()) {
            return $location->contact_email;
        }

        return null;
    }

    public function getLocationEmails(): array
    {
        return array_filter(
            array_unique($this->getManagerEmail(), $this->getContactEmail());
        )
    }
```

then, you can just do this

```

With the above, you can then do this:
```
        return $this->markdown('emails.snapshots.month')
            ->to($market->getLocationEmails())
/// ...
```

Also, by adding the functions, you can now easily use a partial mock for the Market and intercept the `getLocationEmails()` method to return anything you want.

Reply:
@pdbreen Thank you for the suggestion! I will open a separate PR and give this route a go.